@xen-orchestra/rest-api/src/open-api/oa-examples/backup-repository.oa-example.mts

```diff
@@ -0,0 +1,47 @@
+export const backupRepositoryIds = [
+  '/rest/v0/backup-repositories/7497c970-6780-4462-a452-fcb8a406ee64',
+  '/rest/v0/backup-repositories/f681cef1-617e-4650-ac31-ffdaead076bf',
+  '/rest/v0/backup-repositories/a72c53c1-eae1-43f6-be6d-7b0342a1f218',
+  '/rest/v0/backup-repositories/98f5cd17-7594-40b8-906d-51cb57dc7b23',
+]
+
+export const partialBackupRepositories = [
+  {
+    id: '7497c970-6780-4462-a452-fcb8a406ee64',
+    name: 'test',
+    url: 'nfs://localhost:80:/path/to/backup?useVhdDirectory=true&encryptionKey=%22obfuscated-q3oi6d9X8uenGvdLnHk2%22',
+    href: '/rest/v0/backup-repositories/7497c970-6780-4462-a452-fcb8a406ee64',
+  },
```

Comment:
Your partial example must match your default filter/fields.
If you only add `id, name` in `fields`, you should only display `id, name, href` in your examples.